webui_runner/plugs/lucet_app/polybench/build.sh

```diff
@@ -0,0 +1,107 @@
+./medley/deriche/deriche.c
+./medley/floyd-warshall/floyd_warshall.c
+./medley/nussinov/nussinov.c
+./stencils/adi/adi.c
+./stencils/fdtd-2d/fdtd_2d.c
+./stencils/heat-3d/heat_3d.c
+./stencils/jacobi-1d/jacobi_1d.c
+./stencils/jacobi-2d/jacobi_2d.c
+./stencils/seidel-2d/seidel_2d.c
+)
+
+polyfiles2=(
+./datamining/correlation/correlation.c
+)
+
+
+#Build Polybench
+mkdir -p lucet_app_compile/
+
+# Compile utilities/polybench.c
+echo ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} -I utilities -c  utilities/polybench.c -o lucet_app_compile/polybench.wasm.o
+${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} -I utilities -c  utilities/polybench.c -o lucet_app_compile/polybench.wasm.o
+
+
+# Compile the rest
+for cfile in ${polyfiles[@]}; do
+		echo "*****"
+    echo ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} -I utilities $PRINT_TIME -c $cfile -o $(basename -s .c "$cfile").wasm.o
+    ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} -I utilities $PRINT_TIME -c $cfile -o $(basename -s .c "$cfile").wasm.o
+		echo "++++++"
+    echo ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} $(basename -s .c "$cfile").wasm.o lucet_app_compile/polybench.wasm.o -o $(basename -s .c "$cfile").wasm  -nostartfiles -Wl,--no-entry -Wl,--export-all -Wl,--gc-sections 
+          ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} $(basename -s .c "$cfile").wasm.o lucet_app_compile/polybench.wasm.o -o $(basename -s .c "$cfile").wasm -nostartfiles -Wl,--no-entry -Wl,--export-all -Wl,--gc-sections
+		echo "------"
+    echo ${LUCETC} ${LUCETC_FLAGS} --emit=obj  \
+		--bindings=${LUCET_BINDINGS} \
+		--builtins=${LUCET_LIBBUILTINS} $(basename -s .c "$cfile").wasm -o $(basename -s .c "$cfile").wasm
+    ${LUCETC} ${LUCETC_FLAGS} --emit=obj  \
+		--bindings=${LUCET_BINDINGS} \
+		--builtins=${LUCET_LIBBUILTINS} $(basename -s .c "$cfile").wasm -o $(basename -s .c "$cfile").wasm
+		echo "======"
+    echo ${CC} -rdynamic -shared \
+		-L $(dirname ${LUCET_LIBBUILTINS}) \
+		-Wl,-rpath $(dirname ${LUCET_LIBBUILTINS}) \
+		$(basename -s .c "$cfile").wasm -lbuiltins -o $(basename -s .c "$cfile").wasm.so
+
+        ${CC} -rdynamic -shared \
+		-L $(dirname ${LUCET_LIBBUILTINS}) \
+		-Wl,-rpath $(dirname ${LUCET_LIBBUILTINS}) \
+		$(basename -s .c "$cfile").wasm -lbuiltins -o $(basename -s .c "$cfile").wasm.so
+done
+
+
+
+# Build implementation.so
+mkdir -p ${SCRIPT_LOC}/bin; cd ${SCRIPT_LOC}/bin
+>&2 echo ${CC} ${COMMON_CFLAGS} -DWORKLOAD_LOCATION=${SCRIPT_LOC}/benchmark -DVM_LOCATION=${LUCET_APP_ROOT}/build/lucet/target/release/lucet-wasi -shared -o implementation.so ../wrapper.c 
+${CC} -fPIC ${COMMON_CFLAGS} -DWORKLOAD_LOCATION=${SCRIPT_LOC}/benchmark -DVM_LOCATION=${LUCET_APP_ROOT}/build/lucet/target/release/lucet-wasi -shared -o implementation.so ../wrapper.c 
+
+cd ${ORIG_CWD}
```

Comment:
So we need to have separate build scripts for every runtime-suite combination because they each need specific build tweaks?